SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -23,14 +23,32 @@ struct ConfigureConnection
 		: Config(InConfig)
 		, Params()
 		, WorkerType(*Config.WorkerType)
-		, ProtocolLogPrefix(*FormatProtocolPrefix())
+		, WorkerLogPrefix(*FormatLogFilePrefix())
 	{
 		Params = Worker_DefaultConnectionParameters();
 
 		Params.worker_type = WorkerType.Get();
 
-		Params.enable_protocol_logging_at_startup = Config.EnableProtocolLoggingAtStartup;
-		Params.protocol_logging.log_prefix = ProtocolLogPrefix.Get();
+		Logsink.logsink_type = WORKER_LOGSINK_TYPE_ROTATING_FILE;
```

Comment:
Rotating files are useless, what other options are there?

Reply:
Stdout, stderr (both with optional ANSI colour coding), or a custom callback. The Worker SDK currently only supports a rotating log file for actual log writing.

Reply:
One idea would be to create a file stream and direct the WorkerSDK protocol logs to it. This way we could keep the logs in a location of our choosing such as where the `Runtime.log` and `Launch.log` live (spatial/logs/localdeployment/timestamp)